scanner/src/main/kotlin/utils/FileListResolver.kt

```diff
@@ -64,7 +65,7 @@ private val IGNORED_DIRECTORY_MATCHER by lazy {
 
 private fun createFileList(dir: File): FileList {
     val files = dir.walk().onEnter {
-        !IGNORED_DIRECTORY_MATCHER.matches(it.relativeTo(dir).invariantSeparatorsPath)
+        !IGNORED_DIRECTORY_MATCHER.matches(it.relativeTo(dir).invariantSeparatorsPath) && !it.isSymbolicLink()
     }.filter {
         it.isFile
```

Comment:
What about symbolic links to files? I guess these should also not be followed: Either they point to a file inside the tree, and the the target file already gets hashed, or they point to outside the tree, in which case they should probably be ignored.

Reply:
Agree.